android/src/UsbSerialPort.java

```diff
@@ -41,10 +41,17 @@
   public UsbSerialPort(UsbDevice device,int baud) {
     _UsbDevice = device;
     _baudRate = baud;
+    _UsbInterface = -1;
+  }
+  public UsbSerialPort(UsbDevice device,int baud,int iface) {
+    _UsbDevice = device;
+    _baudRate = baud;
+    _UsbInterface = iface;
   }
```

Comment:
Redundant code; both constructors can be merged into one.

Reply:
I'm not familiar with java. Afai read there are no default parameters, so how would you avoid having two constructors?
Or is this what you suggest?
```
public UsbSerialPort(UsbDevice device,int baud) {
     UsbSerialPort(device, baud, -1);
  }
public UsbSerialPort(UsbDevice device,int baud,int iface) {
    _UsbDevice = device;
    _baudRate = baud;
    _UsbInterface = iface;
  }
```

Reply:
Yes, I'd prefer that over having two ctors with redundant code.